sdk/ml/azure-ai-ml/azure/ai/ml/entities/_endpoint/online_endpoint.py

```diff
@@ -39,31 +39,35 @@
     :paramtype tags: typing.Optional[typing.Dict[str, typing.Any]]
-    :keyword properties: The asset property dictionary, defaults to None
+    :keyword properties: The asset property dictionary. If not set, properties defaults to None.
     :paramtype properties: typing.Optional[typing.Dict[str, typing.Any]]
-    :keyword auth_mode: Possible values include: "aml_token", "key", defaults to KEY
+    :keyword auth_mode: Possible values include: "aml_token", "key". If not set, auth_mode defaults to KEY.
     :type auth_mode: typing.Optional[str]
-    :keyword description: Description of the inference endpoint, defaults to None
+    :keyword description: Description of the inference endpoint. If not set, description defaults to None.
     :paramtype description: typing.Optional[str]
-    :keyword location: Location of the resource, defaults to None
+    :keyword location: The geo-location of where the resource lives. If not set, location defaults to None.
     :paramtype location: typing.Optional[str]
-    :keyword traffic: Traffic rules on how the traffic will be routed across deployments, defaults to None
+    :keyword traffic: Percentage of traffic from endpoint to divert to each deployment. Traffic values
+        need to sum to 100. If not set, traffic defaults to None.
     :paramtype traffic: typing.Optional[typing.Dict[str, int]]
-    :keyword mirror_traffic: Duplicated live traffic used to inference a single deployment, defaults to None
+    :keyword mirror_traffic: Percentage of traffic to be mirrored to each deployment without using returned scoring.
+        Traffic values need to sum to utmost 50. If not set, mirror_traffic defaults to None.
     :paramtype mirror_traffic: typing.Optional[typing.Dict[str, int]]
-    :keyword identity: Identity Configuration, defaults to SystemAssigned
+    :keyword identity: Identity Configuration. If not set, identity defaults to SystemAssigned.
     :paramtype identity: typing.Optional[IdentityConfiguration]
-    :keyword scoring_uri: Scoring URI, defaults to None
+    :keyword scoring_uri: Scoring URI. If not set, scoring_uri defaults to None.
     :paramtype scoring_uri: typing.Optional[str]
-    :keyword openapi_uri: OpenAPI URI, defaults to None
+    :keyword openapi_uri: OpenAPI URI. If not set, openapi_uri defaults to None.
     :paramtype openapi_uri: typing.Optional[str]
-    :keyword provisioning_state: Provisioning state of an endpoint, defaults to None
+    :keyword provisioning_state: Provisioning state of an endpoint. Possible values include:
+        "creating", "deleting", "succeeded", "failed", "updating", "canceled".
+        If not set, provisioning_state defaults to None
     :paramtype provisioning_state: typing.Optional[str]
-    :keyword kind: Kind of the resource, we have two kinds: K8s and Managed online endpoints, defaults to None
+    :keyword kind: Kind of the resource, we have two kinds: K8s and Managed online endpoints. If not set, kind defaults to None.
```

Comment:
What value are we expecting here? Are "K8s" and "Managed online endpoints" the strings we're accepting?

Reply:
Hmm okay. 

1. Let's remove the part of the description that mentions the two kinds since we aren't sure
2. Find the codeowner and create a bug on them to update the keyword docstring will a clear explanation of this keyword and what the accepted values are